libs/domain/content/link/link.styles.ts

```diff
@@ -0,0 +1,15 @@
+import { css } from 'lit';
+
+export const styles = css`
+  :host([appearance='dropdown']) {
+    --oryx-button-color: var(--oryx-color-neutral-12);
+
+    padding: 9px 5px;
+    display: flex;
+    height: 24px;
+  }
+
+  :host([appearance='dropdown']:hover) {
+    background-color: var(--oryx-color-neutral-3);
+  }
```

Comment:
We like those styles to be applied to the link component.
We like to try to make them generic enough so they can serve more navigation items in other places (see other comments).